README.md

```diff
@@ -0,0 +1,158 @@
+# Spring Boot Example using OpenTracing Java Agent
+
+NOTE: Use of this example currently requires building the agent - clone and `mvn clean install` 
+[this project](https://github.com/objectiser/java-agent). Then obtain the `opentracing-agent.jar` from the
+`opentracing-agent/target` folder.
+
+NOTE: This example is currently dependent upon [this PR](https://github.com/opentracing-contrib/java-web-servlet-filter/pull/11) therefore this version will need to be cloned and built locally.
+
+## The Example
+
+The example services are based on [this example](http://github.com/obsidian-toaster/quick_rest_springboot-tomcat),
+but modified to have two spring boot services, one calling the other.
+
+The first service (Service A) is called by an external client (e.g. curl) using the `/greeting' endpoint, supplying
+an optional `name` parameter. This service then calls Service B to obtain the message template to be used. Finally
+the template, with the optional supplied `name` parameter is returned to the client.
+
+To try out the example, without any instrumentation being added, build each service:
+
+```
+cd servicea
+mvn clean install
+cd ../serviceb
+mvn clean install
+cd ..
+```
+
+then run both services - you will need to start two separate command windows and then run the following
+commands in each:
+
+```
+cd servicea
+mvn spring-boot:run
+```
+
+```
+cd serviceb
+mvn spring-boot:run
+```
+
+Finally in a separate command window run:
+
+```
+curl http://localhost:8080/greeting -d name=Fred
+```
+
+The response from this command should be:
+
+```json
+{"id":2,"content":"Hello, Fred!"}
+```
+
+## OpenTracing Java Agent Instrumentation
+
+### Instrument the technologies and frameworks
+
+The OpenTracing framework integrations and rules are added to the services as dependencies, e.g.
+
+```xml
```

Comment:
Perhaps add a section at the end called "How it works", to explain how it all fits together?